nixos/platform/default.nix

```diff
@@ -298,7 +298,6 @@ in {
         http-connections = 2
         log-lines = 25
         experimental-features = nix-command flakes fetch-closure
```

Comment:
We can also move the other extraOptions over to settings.
It's okay not to do this in this PR, because this keeps the change down to its intended minimal core, and it remains backportable.